locale/de_CH.ts

```diff
@@ -107,18 +107,17 @@ Dies kann zu jeder Zeit über das &quot;Popup-Symbol&quot; in der Werkzeugleiste
       <translation>&lt;h3 &lt;h3 align=&quot;center&quot;&gt;Willkommen zu &lt;b&gt;GoldenDict&lt;/b&gt;&lt;/h2&gt;&lt;p&gt;Das sind die ersten Schritte um mit dem Programm arbeiten zu können. &lt;p&gt; Gehen Sie zunächst auf &lt;b&gt;Bearbeiten » Wörterbücher...&lt;/b&gt; um die Pfade zu abgespeicherten Wörterbüchern hinzuzufügen sowie um Wikipedia und andere Seiten einzurichten oder die Wörterbücher anzuordnen bzw. in Gruppen zu unterteilen.&lt;/p&gt;
 &lt;p&gt;Nun sind Sie bereit das erste Wort nachzuschlagen. Dies können Sie im Eingabe-Fenster machen oder &lt;a href=&quot;Mit Popups arbeiten&quot;&gt;mittels der Popup-Funktion in anderen Applikationen.&lt;/a&gt;&lt;/p&gt;
 &lt;p&gt;Um das Programm anzupassen öffnen Sie &lt;b&gt;Bearbeiten » Einstellungen...&lt;/b&gt;&lt;br /&gt;Viele Einstellungen sind auch mit Tipps versehen.&lt;/p&gt;
-&lt;p&gt;Hilfe, Fragen und Antworten dazu finden Sie im archvierten &lt;a href=&quot;https://github.com/xiaoyifang/goldendict/discussions&quot;&gt;Forum&lt;/a&gt;.&lt;br /&gt;Programm Updates finden sind auf der Webseite &lt;a href=&quot;https://sourceforge.net/projects/goldendict/&quot;&gt;GoldenDict&lt;/a&gt; und den weiterhin gepflegten Source auf &lt;a href=&quot;https://github.com/goldendict/goldendict/&quot;&gt;GoldenDict Source&lt;/a&gt;.
+&lt;p&gt;Hilfe, Fragen und Antworten dazu finden Sie immer noch im archivierten &lt;a href=&quot;http://goldendict.org/forum/&quot;&gt;Forum&lt;/a&gt;.&lt;br /&gt;Ebenso Programm Updates sind auf der Webseite &lt;a href=&quot;https://sourceforge.net/projects/goldendict/&quot;&gt;GoldenDict&lt;/a&gt; und den weiterhin gepflegten &lt;a href=&quot;https://github.com/goldendict/goldendict/&quot;&gt;GoldenDict Source&lt;/a&gt;.
```

Comment:
This URL change is very weird.

Reply:
![image](https://github.com/xiaoyifang/goldendict-ng/assets/20123683/c23ffeaf-b7ee-4fb8-8bb9-1372c942a2c6)

Reply:
This is weird ,I do not think I have changed this.